code/model/Habit.java

```diff
@@ -0,0 +1,91 @@
+package code.model;
+
+/** The habit class
+ * 
+ * @author	CS 3212
+ * @version Spring 2022
+ */
+public class Habit {
+	public static final String NULL_TEXT_ERROR = "text for the habit cannot be null";
+	public static final String EMPTY_TEXT_ERROR = "text for the habit cannot be empty";
+	private String text;
+	private boolean isComplete;
```

Comment:
Recommend changing this to `complete`.
I wrote it down as you would with a getter in the class diagram, that one's on me.